### PR TITLE
PMM-13593 Send pipeline messages to #pmm-notifications

### PR DESCRIPTION
**Pull request by ademidoff**
[PMM-13593](https://perconadev.atlassian.net/browse/PMM-13592)

[PMM-13593]: https://perconadev.atlassian.net/browse/PMM-13593?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pmm/v3/pmm3-release.groovy [hunk 1]
```diff
@@ -485,11 +485,11 @@ ENDSSH
             deleteDir()
         }
         success {
-            slackSend botUser: true, channel: '#pmm-dev', color: '#00FF00', message: "PMM ${VERSION} was released!\nBuild URL: ${BUILD_URL}\n${env.SCAN_REPORT_URL}"
+            slackSend botUser: true, channel: '#pmm-notifications', color: '#00FF00', message: "PMM ${VERSION} was released!\nBuild URL: ${BUILD_URL}\n${env.SCAN_REPORT_URL}"
```
Comment (ademidoff) [inline pmm/v3/pmm3-release.groovy:488]:
```suggestion
            slackSend botUser: true, channel: '#pmm', color: '#00FF00', message: "PMM ${VERSION} was released!\nBuild URL: ${BUILD_URL}\n${env.SCAN_REPORT_URL}"
```

File: pmm/v3/pmm3-release-candidate.groovy [hunk 2]
```diff
@@ -137,7 +137,7 @@ pipeline {
             name: 'REMOVE_RELEASE_BRANCH'
         )
         string(
-            defaultValue: '#pmm-dev',
+            defaultValue: '#pmm-notifications',
```
Comment (ademidoff) [inline pmm/v3/pmm3-release-candidate.groovy:140]:
```suggestion
            defaultValue: '#pmm',
```

**Review by puneet0191 — APPROVED** [review]

File: pmm/v3/pmm3-release-candidate.groovy [hunk 3]
```diff
@@ -372,5 +372,8 @@ Tarball ARM64: ${env.TARBALL_ARM64_URL}
 ${env.SCAN_REPORT_URL}
                       """
         }
+        failure {
+            slackSend botUser: true, channel: env.NOTIFICATION_CHANNEL, color: '#FF0000', message: "[${JOB_NAME}]: RC build failed :fire: - ${BUILD_URL}"
```
Comment (BupycHuk) [inline pmm/v3/pmm3-release-candidate.groovy:376]: I think it should be pmm-internal instead of pmm for failure

File: pmm/v3/pmm3-release.groovy [hunk 4]
```diff
@@ -485,11 +485,11 @@ ENDSSH
             deleteDir()
         }
         success {
-            slackSend botUser: true, channel: '#pmm-dev', color: '#00FF00', message: "PMM ${VERSION} was released!\nBuild URL: ${BUILD_URL}\n${env.SCAN_REPORT_URL}"
+            slackSend botUser: true, channel: '#pmm', color: '#00FF00', message: "PMM ${VERSION} was released!\nBuild URL: ${BUILD_URL}\n${env.SCAN_REPORT_URL}"
             slackSend botUser: true, channel: '#releases', color: '#00FF00', message: "PMM ${VERSION} was released!\nBuild URL: ${BUILD_URL}\n${env.SCAN_REPORT_URL}"
         }
         failure {
-            slackSend botUser: true, channel: '#pmm-ci', color: '#FF0000', message: "[${JOB_NAME}]: release failed - ${BUILD_URL}"
+            slackSend botUser: true, channel: '#pmm-notifications', color: '#FF0000', message: "[${JOB_NAME}]: release failed - ${BUILD_URL}"
```
Comment (BupycHuk) [inline pmm/v3/pmm3-release.groovy:492]: should it be pmm-internal?

Comment (ademidoff) [reply pmm/v3/pmm3-release.groovy]: Done.

File: pmm/v3/pmm3-release-candidate.groovy (hunk 3 above)
Comment (ademidoff) [reply pmm/v3/pmm3-release-candidate.groovy]: Done.

File: pmm/v3/pmm3-release-candidate.groovy [hunk 5]
```diff
@@ -137,7 +137,7 @@ pipeline {
             name: 'REMOVE_RELEASE_BRANCH'
         )
         string(
-            defaultValue: '#pmm-dev',
+            defaultValue: '#pmm',
```
Comment (BupycHuk) [inline pmm/v3/pmm3-release-candidate.groovy:140]: information about RC started should go to pmm-internal instead of pmm

**Review by BupycHuk — APPROVED** [review]

File: pmm/v3/pmm3-release-candidate.groovy (hunk 5 above)
Comment (ademidoff) [reply pmm/v3/pmm3-release-candidate.groovy:140]: Thanks, I missed that one.

**Pull request merged**